stravalib/client.py

```diff
@@ -1566,41 +1687,59 @@ def create_subscription(
     def handle_subscription_callback(
         self, raw, verify_token=model.Subscription.VERIFY_TOKEN_DEFAULT
     ):
-        """
-        Validate callback request and return valid response with challenge.
+        """Validate callback request and return valid response with challenge.
+
+        Parameters
+        ----------
+        raw :
+        verify_token : default=model.Subscription.VERIFY_TOKEN_DEFAULT
+
+        Returns
+        -------
+        Dict[str, str]
+            The JSON response expected by Strava to the challenge request.
 
-        :return: The JSON response expected by Strava to the challenge request.
-        :rtype: Dict[str, str]
         """
         callback = model.SubscriptionCallback.deserialize(raw)
         callback.validate(verify_token)
         response_raw = {"hub.challenge": callback.hub_challenge}
         return response_raw
 
+    # TODO: i'm not sure what raw's "type" is here
```

Comment:
Yes, `raw` is a dict representation of the JSON request. It's good you found this, because it shows a usage of the deprecated `deserialize()` method. I just created #358 as a reminder to get rid of those.

Reply:
ok great! i'll leave a TODO: to mark this! and will update the docstring now.